katib/v1alpha1/base/katib-ui-deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: katib-ui
+  namespace: kubeflow
```

Comment:
can we pass this as a var?

Reply:
it can be overwritten by `namespace` in kustomization.yaml